pkg/syncer/metadatasyncer.go

```diff
@@ -1010,13 +983,21 @@ func ReloadConfiguration(metadataSyncer *metadataSyncInformer, reconnectToVCFrom
 		} else {
 			oldvCenter := metadataSyncer.configInfo.Cfg.Global.VCenterIP
 			newVCConfig, err := cnsvsphere.GetVirtualCenterConfig(ctx, cfg)
-
+			newVCConfig.ReloadVCConfigForNewClient = true
 			if err != nil {
 				return logger.LogNewErrorf(log, "failed to get VirtualCenterConfig. err=%v", err)
 			}
 			if newVCConfig != nil {
+				if oldvCenter != newVCConfig.Host {
+					log.Infof("Observed new vCenter server: %q in the config secret. "+
```

Comment:
This is a single VC case, right.
`Observed new vCenter server` can be updated to `Observed a change in the vCenter server details.`

Reply:
this is a new VC even for single VC case